lib/iotagent-ul.js

```diff
@@ -122,12 +136,15 @@ function start(newConfig, callback) {
  *
  */
 function stop(callback) {
-    logger.info(context, 'Stopping IoT Agent');
+    logger.info(context, 'Stopping IoT Agent: ' + (new Date()));
```

Comment:
Is it really needed to log the current date and time since already included in the `time` property of the message logged by `logops`?